web/src/app/profile/approved-researcher/components/AgreementForm.tsx

```diff
@@ -27,21 +27,20 @@ export default function ApprovedResearcherForm(props: ApprovedResearcherFormProp
   };
 
   return (
-    <div className="form-container">
+    <div className="form form__wrapper">
       {!submitted && (
         <form onSubmit={handleSubmit}>
-          I agree
           <input
             type="checkbox"
             name="agreed"
             onChange={() => {
               setAgreed(!agreed);
             }}
             checked={agreed}
-            className="input-checkbox"
             required
-          />
-          <button type="submit" className="submit-button">
+          />{" "}
+          I agree
+          <button type="submit" className="btn button--submit">
             Submit
```

Comment:
Any reason for the mixing of `btn` and `button` here?

Reply:
habitually used to `btn` as general class, which is your preference and I'll update them